auth_jwt_demo/__manifest__.py

```diff
@@ -5,12 +5,13 @@
     "name": "Auth JWT Test",
     "summary": """
         Test/demo module for auth_jwt.""",
-    "version": "14.0.1.2.0",
+    "version": "15.0.1.0.0",
```

Comment:
Don't change the version yet

Reply:
this was done to avoid pre-commit issues